server/audit_math/bravo.py

```diff
@@ -432,6 +423,7 @@ def get_sample_size(
     if (
         num_ballots > large_election_threshold
         and samples["0.9"]["size"] >= all_ballots_threshold
+        or p_w == p_l
```

Comment:
i think this might error in case that `p_w == p_l` because `samples` won't have key `"0.9"` set.

Reply:
Hmm, it doesn't in the tests, but that's an interesting point. I did this to get around the linter warning that I was returning too many times.